tslint.json

```diff
@@ -104,7 +104,7 @@
     "no-empty": false,
     "no-empty-interface": true,
     "no-eval": true,
-    "no-increment-decrement": true,
+    // "no-increment-decrement": true,
```

Comment:
Good call - left this in when I tried out the microsoft tslint rules but forgot to get rid of it!